webpack.config.js

```diff
@@ -0,0 +1,111 @@
+"use strict";
+
+const CleanWebpackPlugin = require('clean-webpack-plugin')
+  ,BrowserSyncPlugin = require('browser-sync-webpack-plugin')
+  ,HtmlWebPackPlugin = require('html-webpack-plugin')
+  ,MiniCssExtractPlugin = require('mini-css-extract-plugin')
+  ,OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin')
+  ,Package = require('./package.json')
+  ,path = require('path')
+  ,project_config = require('./config.js')
+  ,UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+  ,buildDestination = './dist'
+  ,versionJS = Package.webpack_bundle_version_js
+  ,SassLintPlugin = require('sass-lint-webpack') 
+  ,versionCSS       = Package.webpack_bundle_version_css;
```

Comment:
add a space after the `,` on all of these and either align your equal signs or don't, but be consistent